api-docs/cloud-serverless/ref.yml

```diff
@@ -14680,8 +14680,8 @@ tags:
       | &nbsp;Code&nbsp; | Status                   | Description      |
       |:-----------:|:------------------------ |:--------------------- |
       | `200`       | Success                  |                       |
-      | `201`       | Created                           | Successfully created a resource. The response body may contain details. |
-      | `204`       | No content      | The request succeeded, but InfluxDB doesn't typically return a response body for the operation. |
+      | `201`       | Created                  | Successfully created a resource. The response body may contain details. |
```

Comment:
201 no longer exists in serverless?

Reply:
This table isn't specific to writes--we still have v2 endpoints in Serverless that respond with `201`, correct?

Reply:
In the context of this change, I decided to leave 201. This section of the doc seems to be about the API generally, not the write API specifically.

Reply:
TBH, I thought about whether we do or don't return 201 anywhere, and I chose not to go searching through the IOX and IDPE repos.